### ISSUE TITLE
Issue with Scaffold Generator

### ISSUE THREAD

**Issue by powerdude** [adospace/reactorui-maui#201]
Hi, I'm using the ScaffoldGenerator for some SyncFusion objects and getting error:

```
error CS1061: 'RxAnimation' does not contain a definition for 'CurrentValue' and no accessible extension method 'CurrentValue' accepting a first argument of type 'RxAnimation' could be found (are you missing a using directive or an assembly reference?)
```

 in this method for all classes:

```
 public static T FontSize<T>(this T sfNumericEntry, double fontSize, RxDoubleAnimation? customAnimation = null)
            where T : ISfNumericEntry
        {
            sfNumericEntry.FontSize = fontSize;
            sfNumericEntry.AppendAnimatable(global::Syncfusion.Maui.Inputs.SfNumericEntry.FontSizeProperty, customAnimation ?? new RxDoubleAnimation(fontSize), v => sfNumericEntry.FontSize = v.CurrentValue());
            return sfNumericEntry;
        }
```

**Comment by adospace** [comment]
ops, sorry, it's a bug, The fix is coming asap

**Comment by adospace** [comment]
fixed in 2.0.20
